CHANGELOG.md

```diff
@@ -1,4 +1,83 @@
+8. Implemented support for Monitor Gateway Subnets in **aviatrix_transit_gateway** and **aviatrix_spoke_gateway** with the following attributes:
+  - ``enable_monitor_gateway_subnets``
+  - ``monitor_exclude_list``
+9. Implemented support for private transit gateway peering with single-tunnel mode
+10. Implemented support for transit to external device using IKEv2
+11. Implemented new resource to support transit in azure with express route:
+  - **resource_aviatrix_azure_vng_conn**
+12. Implemented support for private OOB:
+  - New resource **aviatrix_controller_private_oob**
+  - New attributes ``enable_private_oob``, ``oob_management_subnet``, and ``oob_availability_zone`` in **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+  - New attributes ``enable_private_oob_subnet`` in **aviatrix_vpc**
+13. Implemented support for proxy configuration:
+  - New resource: **aviatrix_proxy_config**
+14. Implemented support for OCI provider in **aviatrix_vpc**
+15. Implemented support for Aviatrix client/ovpn file download from the controller when SAML authentication is used:
+  - New resource: **aviatrix_vpn_cert_download**
+16. Implemented new resources to support logging:
+  - **resource_aviatrix_remote_syslog**
+  - **resource_aviatrix_splunk_logging**
+  - **resource_aviatrix_filebeat_forwarder**
+  - **resource_aviatrix_sumologic_forwarder**
+  - **resource_aviatrix_datadog_agent**
+  - **resource_aviatrix_netflow_agent**
+  - **resource_aviatrix_cloudwatch_agent**
+
+### Enhancements:
+1. Added Computed value ``ha_lan_interface_cidr`` in **aviatrix_transit_gateway**
+2. Changed **aviatrix_gateway** attribute ``monitor_exclude_list`` type from String to Set of Strings
+3. Added support of ``tag_list`` for Azure provider in **aviatrix_gateway**, **aviatrix_transit_gateway**, and **aviatrix_spoke_gateway** resources and data sources
+4. Added ``customized_transit_vpc_routes`` in **aviatrix_transit_gateway** resource and data source
+5. Added ``azure_vnet_resource_id`` as output for **aviatrix_vpc** resource and data source
+
+### Bug Fixes:
+1. Fixed issue where users could not create an **aviatrix_firewall_instance** if the VPC/VNET was not managed by the Aviatrix controller
+2. Fixed an argument ordering issue in **aviatrix_site2cloud** Custom Mapped attributes by changing from type Set to List
+3. Fixed race condition when deploying spoke gateway (HA disabled) using ``customized_spoke_vpc_routes`` and ``transit_gw``
+4. Fixed issue where creating **aviatrix_site2cloud** for ActiveActive-enabled gateway causes deltas in state
+5. Fixed issue where attribute ``bgp_manual_spoke_advertise_cidrs`` in **aviatrix_transit_gateway** causes delta in every apply
+6. Fixed issue where Egress Transit Gateway can't be created due to provider block
+7. Fixed issue where an **aviatrix_spoke_gateway** with advertised spoke vpc cidr's can't connect to an **aviatrix_transit_gateway**
```

Comment:
7. Fixed issue where an **aviatrix_spoke_gateway** with advertised spoke VPC CIDRs can't connect to an **aviatrix_transit_gateway**